repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -269,6 +270,12 @@ export interface ResponseWithLocals extends ServerResponse {
     };
 }
 
+export interface PremadeCaps {
+    id: number;
+    contents: DbTimeRestriction[];
```

Comment:
Why are we using DbTimeRestriction here?

Reply:
I think because we dont know what the type will be, for now just use contents: {} and when the linting complains, just use an eslint ignore